src/main/java/com/zufar/onlinestore/payment/api/impl/customer/StripeCustomerProcessor.java

```diff
@@ -0,0 +1,60 @@
+package com.zufar.onlinestore.payment.api.impl.customer;
+
+import com.stripe.exception.StripeException;
+import com.stripe.model.Customer;
+import com.stripe.param.CustomerCreateParams;
+import com.zufar.onlinestore.payment.converter.StripeCustomerConverter;
+import com.zufar.onlinestore.payment.exception.StripeCustomerProcessingException;
+import com.zufar.onlinestore.security.api.SecurityPrincipalProvider;
+import com.zufar.onlinestore.user.entity.UserEntity;
+import com.zufar.onlinestore.user.repository.UserRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * This class is responsible for Stipe customer creation. It needed in order to connect
+ * customer with payment intent, this will ensure the correct management of payment by Stripe API
+ */
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class StripeCustomerProcessor {
+
+    private final SecurityPrincipalProvider securityPrincipalProvider;
+    private final StripeCustomerConverter stripeCustomerConverter;
+    private final UserRepository userRepository;
+
+    public Customer processStripeCustomer() throws StripeCustomerProcessingException {
```

Comment:
remove this throws from method declaration

Reply:
Done